port_drayage_plugin/config/params.yml

```diff
@@ -1,3 +1,3 @@
 stop_speed_epsilon: 1.0
-cmv_id: CARMA_TEST_CMV
-cargo_id: TEST_CARGO
+cmv_id: 123 // Test value; CARMA Streets expects cmv_id to be an integer
```

Comment:
This representation is also a little weird, if this is an ID I imagine it's more like the BSM ID where it's intended to be a raw byte-string. Not to say that we can't integerize it, but it may be more accurate to use an unsigned int to avoid any weirdness or misinterpretation. I'm not 100% sure what the interpretation is on CARMA streets side, so I could be off base though.